src/utils/command-utils.js

```diff
@@ -0,0 +1,48 @@
+import chalk from 'chalk'
+
+export function getCommmandSingleFlag(flag, commandLine) {
+  return commandLine.indexOf(flag) > -1
+}
+
+export function getCommandParameter(commandFlag, commandFlagLong, commandLine, commandExample = '') {
+  let parameterValueFlagIndex = commandLine.indexOf(commandFlagLong)
+
+  if (parameterValueFlagIndex === -1)
+    parameterValueFlagIndex = commandLine.indexOf(commandFlag)
+
+  if (parameterValueFlagIndex === -1) {
+    console.log(
+      chalk.redBright(`'${commandFlag || commandFlagLong}' flag is missing (eg '${commandExample}')`)
```

Comment:
It might be better to show the entire `eg` text only if `commandExample` is available.